src/plugin-slots/MobileUserMenuToggleSlot/README.md

```diff
@@ -0,0 +1,76 @@
+# Learning User Menu Toggle Slot
+
+### Slot ID: `org.openedx.frontend.layout.header_mobile_user_menu_trigger.v1`
+
+### Slot ID Aliases
+* `learning_mobile_user_menu_trigger_slot`
+
+## Description
+
+This slot is used to replace/modify/hide the contents of the user menu toggle button.
+
+## Examples
+
+### Modify Avatar
+
+The following `env.config.jsx` will modify the icon for the learning user menu toggle button. **Note:** The icon is only shown on mobile screens.
```

Comment:
```suggestion
The following `env.config.jsx` will modify the icon for the mobile user menu toggle button.
```

I think mentioning the icon being mobile-only might make it *less* clear that the entire slot is only rendered on mobile (also this isn't the learning menu)